main.py

```diff
@@ -115,6 +115,14 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: string. The route to be redirected.
```

Comment:
`string` --> `str`.

Reply:
Hey @DubeySandeep , I have worked on this.